tests/ga/test_extension.py

```diff
@@ -541,16 +541,29 @@ def test_ext_handler_rollingupgrade(self, *args):
         self._assert_ext_status(protocol.report_ext_status, "success", 0)
 
     @patch('azurelinuxagent.ga.exthandlers.add_event')
-    def test_ext_handler_download_failure(self, mock_add_event, *args):
+    def test_ext_handler_download_failure_transient(self, mock_add_event, *args):
```

Comment:
Is there a corresponding test case for ReportStatusExtended?

Reply:
Will add.